editor.planx.uk/src/@planx/components/Page/model.ts

```diff
@@ -1,10 +1,27 @@
+import { cloneDeep } from "lodash";
+
 import { MoreInformation, parseMoreInformation } from "../shared";
+import { Schema } from "../shared/Schema/model";
+import { PAGE_SCHEMAS } from "./Editor";
+
+export interface PageSchema extends Schema {
+  min: 1;
+  max: 1;
+}
```

Comment:
This enforces that Pages can only have one item, whilst allowing us to re-use all logic from `useSchema()`